doc/src/manual/style-guide.md

```diff
@@ -447,3 +447,43 @@ julia> h(1)
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
 in order to make it easier to use your code.
+
+## Workarounds for named positional arguments
+Named positional arguments are not supported in Julia, however, some users may still wish to use a named positional arguments code-style.
```

Comment:
I would include more discussion about what a "named positional argument" is in the first place, with maybe an explicit example of "some languages allow you to call `f(x,y,z)` by `f(x=x, y=y, z=z)` .... "